<?xml version="1.0"?>
<launch> 

  <node name="lidar_scan_filtering_node" pkg="spinning_lidar_utils" type="lidar_scan_filtering" output="screen">
    <param name="min_dist_to_sensor" value="0.5" />
    <param name="laser_link" type="string" value="laser_mount_base_link" />
    <param name="laser_scan_topic" type="string" value="spinning_lidar/scan" />
    <param name="filtered_scan_topic" type="string" value="spinning_lidar/filtered_scan" />
    <!-- <param name="filtered_cloud_topic" type="string" value="spinning_lidar/filtered_cloud" /> -->
    <param name="filtered_cloud_topic" type="string" value="sync_scan_cloud_filtered" />
  </node>


</launch>